src/BAP/SimpleBTSBundle/Form/EventListener/IssueSubscriber.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace BAP\SimpleBTSBundle\Form\EventListener;
+
+use BAP\SimpleBTSBundle\Entity\Issue;
+use BAP\SimpleBTSBundle\Entity\IssueResolution;
+use Doctrine\ORM\EntityManager;
+use Symfony\Component\Form\FormEvent;
+use Symfony\Component\Form\FormEvents;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class IssueSubscriber implements EventSubscriberInterface
+{
+    /**
+     * @var EntityManager
+     */
+    protected $entityManager;
+
+    /**
+     * @param EntityManager $entityManager
+     */
+    public function __construct(EntityManager $entityManager)
```

Comment:
Please try to not use EntityManaget. You can use Doctrine\Common\Persistence\ManagerRegistry here